src/css/style.css

```diff
@@ -0,0 +1,74 @@
+@import url("./helpers/reset.css");
+@import url("./helpers/variables.css");
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: var(--fz);
+  font-style: normal;
+  font-weight: 500;
+  color: var(--black);
+  margin: 0;
+  line-height: 100%;
+}
+
+.container {
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.nav {
+  min-width: 635.54px;
+}
```

Comment:
You don't need to specify min-width in this case
Also don't use float values in your styles like "635.54px"